src/Public/css/sis.css

```diff
@@ -85,26 +85,38 @@ fieldset { padding:0; border:0; margin-top:25px; }
   margin-bottom: .5em;
   border-collapse: separate;
   max-width: 250px;
-  
+
 }
 .selbtn {
   height: 35px;
-  width: 35px;
-  background-color: #eee;
+  min-width: 30px;
+  background-color: #ddd;
+  text-align: center;
+  margin: 5px;
+  font-size: 1.2em;
+  padding: 5px 0;
+  color: #fff;
+  display: inline-block;
 }
 .poweredon { cursor:pointer; }
-.s1on, .s2on, .s3on, .s4on {
+.s1on, .s2on, .s3on, .s4on, .s8on {
```

Comment:
something to perhaps consider for future - redo the class/id of this stuff, this one in particular seems perfect for a class, rather than an ID

Reply:
Erm, not quite sure what you mean? Those are classes?

Reply:
I can CSS. In which case, why does each "station" have its own class? Should either be IDs, or more generically named (`powered`, or whatever)